src/SourceGenerators/System.Xaml/System.Xaml/XamlXmlReader.cs

```diff
@@ -561,12 +561,12 @@ StartTagInfo GetStartTagInfo ()
 						break;
 
 					default:
-						if (IsIgnored(r.Prefix, r.NamespaceURI, out var shouldTreatAsDefaultNamespace))
+						if (IsIgnored(r.Prefix, r.NamespaceURI))
```

Comment:
I must have this code as a way to fix a scenario that wasn't compiling, but I don't see know what this scenario is. Probably it was fixed later in a different way that made this code not required any more, and this code here doesn't work well for attached properties when using XAML-included namespaces.

Reply:
Yes, it was there for a reason. `When_Conditional_IsApiContractPresent` failed with this change. I made additional changes in f3b9f2e. Let's see if it will pass.